src/containers/Sidebar.js

```diff
@@ -70,8 +122,10 @@ export class Sidebar extends Component {
 
     const defaultLinks = _.keys(defaults);
     let hiddenLinks;
+
     try {
       hiddenLinks = config.jekyll_admin.hidden_links;
+      hiddenLinks = hiddenLinks || [];
```

Comment:
Let's move default array assignment to upper line.

Reply:
If I have it as:
```js
hiddenLinks = [];
hiddenLinks = config.jekyll_admin.hidden_links;
```
Then I'll have to add additional logic to test if `config.jekyll_admin.hidden_links` is `null`

Reply:
```js
hiddenLinks = config.jekyll_admin.hidden_links || [];
```

Reply:
Of course..!! :man_facepalming: